apps/api/src/app/workflows-v2/shared/schemas/digest-control.schema.ts

```diff
@@ -67,7 +67,7 @@ export const digestUiSchema: UiSchema = {
     },
     digestKey: {
       component: UiComponentEnum.DIGEST_KEY,
-      placeholder: null,
+      placeholder: '',
```

Comment:
`0` and `negative` values as amount are not returning any validation error - we should add `min(1)` validation to the amount and also to be consistent with delay:

![image](https://github.com/user-attachments/assets/9870cd47-d2ba-456d-9803-65a7e1052861)

![image](https://github.com/user-attachments/assets/d35845f4-8a49-4c1a-a098-51c366bcab6c)

Reply:
These values should raise issues from the API but also not be allowed at all from the UI in the numeric input. That way, we have a clean API and a UI that guides the user to the right values.

Reply:
will add on both sides of the stack